plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -524,7 +524,9 @@ func ClusterRoles() []rbacv1.ClusterRole {
 		Rules: []rbacv1.PolicyRule{
 			rbacv1helpers.NewRule("get").URLs(
 				"/.well-known/openid-configuration",
+				"/.well-known/openid-configuration/",
```

Comment:
If we do decide to update the RBAC rules, this should have a comment that describes why we need the `/` terminated endpoints here.

Reply:
Sorry for the late, comment added